backend/src/main/java/quarano/department/web/TrackedCaseController.java

```diff
@@ -109,6 +110,10 @@
 
 	HttpEntity<?> createTrackedCase(TrackedCaseDto payload, CaseType type, Department department, Errors errors) {
 
+		if (payload.getZipCode() != null) {
+			checkZipCodeMatchRKI(payload.getZipCode(), errors);
+		}
```

Comment:
In `TrackedCaseRepresentations.validate` is the validation of a TrackedCase. This is called by `representations.from`.

We should keep the validations together.